index.js

```diff
@@ -48,7 +53,7 @@ Merry.prototype.defaultRoute = function (handler) {
 
   function routeHandler (req, res, params) {
     var ctx = new Ctx(req, res, self.log)
-    ctx.params = params
+    ctx.params = params.params
```

Comment:
We're doing a double object allocation here; I propose we reuse the object returned from the router instead - should save some on GC